client/src/components/ProjectItem.tsx

```diff
@@ -18,24 +18,26 @@ const ProjectItem: React.FC<ProjectItemProps> = ({
     techList
 }) => {
     return (
-        <div className="project-card rounded p-8 m-6 max-w-xs rign-2 ring-offset-2 ring-stone-900 shadow-2xl shadow-gray-400/75 hover:shadow-sky-500">
-            <a href={liveSiteLink}>
-                <div className="project-info rounded">
-                    <h3 className="project-title">{name}</h3>
-                    <p className="project-desc text-sm italic  pb-5">{summary}</p>
-                    <img className="rounded boarder-2 border-stone-900" src={imgSrc} alt={name} />
+        <div className="project-card rounded p-8 m-6 max-w-xs ring-2 ring-offset-2 ring-stone-900 shadow-2xl shadow-gray-400/75 hover:shadow-sky-500 hover:scale-105 transform transition-transform duration-300">
+            <a href={liveSiteLink} className="flex flex-col h-full">
+                <div className="project-info rounded flex-1">
+                    <h3 className="project-title text-lg title-font text-center">{name}</h3>
+                    <p className="project-desc text-sm italic pb-5">{summary}</p>
+                    <img className="rounded border-2 mb-4" src={imgSrc} alt={name} />
                     <div className="flex flex-wrap justify-center py-1">
                         {techList.map((tech, index) => (
-                            <div key={index}><span className="box-decoration-clone bg-gradient-to-r from-yellow-600 to-sky-700 px-2 mx-2 rounded-lg">{tech}</span></div>
+                            <span key={index} className="text-white bg-sky-700 px-2 py-1 ml-2 mb-1 mt-1 rounded-md block">
+                                {tech}
+                            </span>
                         ))}
                     </div>
                 </div>
+                <div className="flex justify-center text-skyl-400 py-3">
+                    <a href={sourceCodeLink} className="mt-auto title-font">&lt; SOURCE CODE/ &gt;</a>
+                </div>
             </a>
```

Comment:
<a> tags can't wrap <a> tags. In other terms, and <a> tag cannot be a descendant of an <a> tag.

Reply:
Looks good now.